app/index.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-import React from 'react';
+import * as React from 'react';
```

Comment:
This is the latest recommended way to import React, I'll probably change them all when I upgrade flow.